pkgs/development/libraries/olm/default.nix

```
@@ -70,6 +80,11 @@ stdenv.mkDerivation rec {
         project lead:
         <https://matrix.org/blog/2024/08/16/this-week-in-matrix-2024-08-16/#dept-of-encryption-closed-lock-with-key>
 
+      * A blog post about libolm's deprecation on the Matrix.org blog that also
+        explains why upstream believes the vulnerabilities are not exploitable
+        over the network:
+        <https://matrix.org/blog/2024/08/libolm-deprecation/>
```

I would remove the TWIM link (since it is included in this) and make this just:

```suggestion
      * The statement about the deprecation and vulnerabilities from the
        Matrix.org Foundation:
        <https://matrix.org/blog/2024/08/libolm-deprecation/>
```

as the earlier paragraph will already give the context of upstream’s view on the matter.